source/guides/models/repositories.md

```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
   # => nil
 ```
 
+## Saving more than one record
+
+For saving bunch of records you need to create `#create_many` command:
```

Comment:
How about:

For saving bunch of records you need to **define a**`#create_many` command:

create #create_many looks a little strange to me :)